playzero/zero-network
# Generate chain specs and genesis for dev and stage

Genesis (Subzero runtime):
- Balances
- Tokens
- Control

chain_spec (Subzero runtime):
- DEV
- STAGE

Test scenario:
- Check if all endowed accounts have expected balances
- Check if all orgs were created with proper treasury addresses
- Check if all collators have proper keys
- Build ok
- Test ok

**!Rebase and merge!**
Commit was used to tag a docker image.
